src/psyclone/dynamo0p3.py

```diff
@@ -5700,7 +5827,7 @@ def _cell_ref_name(self):
         '''utility routine which determines whether to return the cell value
         or the colourmap lookup value '''
         if self._kern.is_coloured():
-            return "cmap(colour, cell)"
+            return self._kern.colourmap + "(colour, cell)"
```

Comment:
plylint complains about the else after this statement. I know you've not changed that line but I thought it was close enough :-)

Reply:
:-) Done.

Reply:
Great, thanks.